extension/src/paymentHandler/payment-utils.js

```diff
@@ -104,11 +104,12 @@ function createChangeTransactionStateAction (transactionId, transactionState) {
   }
 }
 
-function createSetCustomFieldAction (name, value) {
+function createSetCustomFieldAction (name, response) {
   return {
     action: 'setCustomField',
     name,
-    value
+    // todo (ahmet): Don't stringify here. Sometimes you want to pass JSON object, e.g. for LocalizedEnum.
```

Comment:
I have seen a couple of TODOs in this PR. Why not addressing them right away?

Reply:
I removed that todo, as all request and responses will be passed as an object to those utilities. https://github.com/commercetools/commercetools-adyen-integration/pull/232#discussion_r406259171